CMakeLists.txt

```diff
@@ -489,57 +486,24 @@ if(UNIX)
     set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/bin")
 endif(UNIX)
 
-    if(UNIX)
-
-        if(ANTILIB_PATH)
-            ADD_DEFINITIONS(-DANTILIB_PATH="${ANTILIB_PATH}")
-
-            # Generally package manager takes care of this while installing the new library, but not always
-            # especially with cmake
-            ADD_DEFINITIONS(-DLD_LIBRARY_PATH="${ANTILIB_PATH}:${LD_LIBRARY_PATH}")
-
-            # those instructions have to be before add_executable() or add_library() is called, otherwise it has no effect
-            set(CMAKE_INSTALL_RPATH "${ANTILIB_PATH}")
-            set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-        endif()
-
-            add_library( antilib
-                SHARED
-                ${antimicrox_HEADERS_MOC}
-                ${antimicrox_SOURCES}
-                ${antimicrox_FORMS_HEADERS}
-                ${antimicrox_RESOURCES_RCC}
-            )
-
-            set_target_properties(antilib PROPERTIES
-                SOVERSION ${ANTILIB_VERSION}
-            )
-
-
-            #target_link_libraries (antilib Qt5::Widgets Qt5::Core Qt5::Test Qt5::Gui Qt5::Network Qt5::Concurrent ${SDL_LIBRARY} ${LIBS})
-            target_link_libraries (antilib Qt5::Widgets Qt5::Core Qt5::Gui Qt5::Network Qt5::Concurrent ${SDL_LIBRARY} ${LIBS})
-
-            if (WITH_X11)
-                target_link_libraries(antilib Qt5::X11Extras)
-            endif()
+if(UNIX)
+        add_executable(antimicrox
+            ${antimicrox_MAIN}
+            ${antimicrox_HEADERS_MOC}
+            ${antimicrox_SOURCES}
+            ${antimicrox_FORMS_HEADERS}
+            ${antimicrox_RESOURCES_RCC}
+        )
 
 
-            add_executable(antimicrox ${antimicrox_MAIN})
-            target_link_libraries (antimicrox antilib Qt5::Widgets Qt5::Core Qt5::Gui Qt5::Network Qt5::Concurrent)
+        target_link_libraries(antimicrox Qt5::Widgets Qt5::Core Qt5::Gui Qt5::Network Qt5::Concurrent ${LIBS})
```

Comment:
Where's `${SDL_LIBRARY}`? Maybe _that_ contained `xcb`?

Reply:
`pkg_check_modules(XXX REQUIRED yyy)` provides `XXX_LIBRARIES` which in the case is `SDL2_LIBRARIES`

this is a oddity in the ways we find libraries.
we could just use `find_package(SDL2 REQUIRED)` which provides `SDL_LIBRARIES` and `SDL_LIBRARY`.

So i think in a previous version we used `find_package` but later it was changed to `pkg_check_modules` but they forgot to remove the variable.(I can't think of any reasons other than older cmake versions behaving differently)

anyways, I could refactor further and bring back `find_package`?

Reply:
and before my changes it was empty so removing it wont change anything.